trivy-image-scan/action.yml

```diff
@@ -34,46 +34,32 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: clean env variables
-      shell: bash
-      run: |
-        echo "TRIVY_FORMAT=" >> $GITHUB_ENV
-        echo "TRIVY_OUTPUT=" >> $GITHUB_ENV
-        echo "TRIVY_IMAGE_TAG=" >> $GITHUB_ENV
-    
     - uses: gradle/gradle-build-action@v2
```

Comment:
nit - if we don't have an input tag, we can skip this to avoid setting up gradle

Reply:
done.